app/controllers/admin/admin_pages_controller.rb

```diff
@@ -11,7 +11,7 @@ def index
   def upgrade
     user = User.find_by id: params[:id]
     if user
-      switch_role user
+      User.switch_role user
```

Comment:
```suggestion
      user.switch_role 
```
switch_role viết instance method nhé

Reply:
em đã fix